docs/calamari/Partnership.md

```diff
@@ -1 +1,143 @@
+
+Web3Go is an all-in-one open data analytics and service platform where everyone can grasp the value behind blockchain data.
+
+## Collators
+
+### P2P
+
+[Twitter](https://twitter.com/P2Pvalidator) | [Website](https://p2p.org/)
+
+P2P provides secure non-custodial staking services for professional investors, allowing token holders to participate in staking without the heavy lifting of running a node.
+
+### Brightlystake
+
+[Twitter](https://twitter.com/brightlystake) | [Website](https://brightlystake.com/)
+
+Brightlystake is a reliable and friendly non-custodial staking service providers.
+
+### Polkadotters
+
+[Twitter](https://twitter.com/Polkadotters1) | [Website](https://polkadotters.medium.com/)
+
+Polkadotters is a founding member of the DOT Validator Alliance whose aim is to help to decentralize Polkadot network and push small validators forward.
+
+### StakeBaby
+
+[Website](https://medium.com/stakebaby)
+
+StakeBaby is a staking and analytics dashboards for Polkadot and Kusama Parachains.
+
+### StakerSpace
+
+[Twitter](https://twitter.com/StakerSpace) | [Website](https://staker.space/)
+
+StakerSpace is a independent secure-validator.
+
+### Pathrock Network
+
+[Twitter](https://twitter.com/pathrock2) | [Website](https://pathrocknetwork.org/)
+
+Pathrock Network is a secure Non-Custodial Staking for PoS Blockchain Projects.
+
+### Let's Node
+
+[Twitter](https://twitter.com/Lets_node) 
+
+Staking providers, educational and informational channel about nodes, testnets and technical assignments.
+
+### CertHum
+
+[Twitter](https://twitter.com/CertHum) | [Website](https://www.certhum.com/)
+
+CertHum provides Web3 infrastructure and is building for a decentralized future.
+
+## Parachains
+
+### Moonriver
+
+[Twitter](https://twitter.com/MoonriverNW) | [Website](https://moonbeam.network/networks/moonriver/)
+
+Moonriver is a companion network to Moonbeam and provides a permanently incentivized canary network. New code ships to Moonriver first, where it can be tested and verified under real economic conditions. Once proven, the same code ships to Moonbeam on Polkadot.
+
+### Karura
+
+[Twitter](https://twitter.com/KaruraNetwork) | [Website](https://acala.network/karura)
+
+Karura is a scalable, EVM-compatible network optimized for DeFi.
+
+### Khala
+
+[Twitter](https://twitter.com/PhalaNetwork) | [Website](https://www.phala.network/en/khala/)
+
+Khala Network is a Kusama Parachain, Phala’s Canary Network.
+
+### Kusama
+
+[Twitter](https://twitter.com/kusamanetwork) | [Website](https://kusama.network/)
+
+Kusama is a network built as a risk-taking, fast-moving ‘canary in the coal mine’ for its cousin Polkadot.
+
+### RMRK
+
+[Twitter](https://twitter.com/RmrkApp) | [Website](https://www.rmrk.app/)
+
+RMRK is a set of NFT legos that give NFTs infinite extensibility, hosted on the Kusama blockchain, Polkadot's canary network, without the need for parachains or smart contracts.
```

Comment:
We are not connected to RMRK nor do we have the asset registered.
We're connected with Statemine (where RMRK lives)

Reply:
i think this doc is about product partnerships, not currently existing integrations